### ISSUE TITLE
Skipping keys with open_dataset ?

### ISSUE THREAD

**Issue by Balinus** [JuliaDataCubes/YAXArrays.jl#465]
Is there a way to skip a variable/keys/etc when using open_dataset?

Right now, I have badly formatted netcdf file, where there is no information on one of the coordinates (nbnds):

![image](https://github.com/user-attachments/assets/4f4b8bd0-429b-4185-84a9-c013ccb1f30e)

I am not able to open the file with `open_dataset` though and I was wondering if it was possible to ignore some the of variables or something similar:

![image](https://github.com/user-attachments/assets/c05ed399-5783-489b-8ca3-4daeaa550316)

**Comment by Balinus** [comment]
output from NCDatasets.jl:

```julia
Group: /

Dimensions
   forecast_date = 1
   lead = 4
   lat = 258
   lon = 480
   quantiles = 19
   nbnds = 2

Variables
  forecast_date   (1)
    Datatype:    DateTime (Int64)
    Dimensions:  forecast_date
    Attributes:
     long_name            = Forecast Creation Date
     units                = days since 2024-03-15
     calendar             = proleptic_gregorian

  lead   (4)
    Datatype:    Int64 (Int64)
    Dimensions:  lead
    Attributes:
     long_name            = Forecast Lead
     units                = months
     bounds               = lead_bnds

  lat   (258)
    Datatype:    Union{Missing, Float32} (Float32)
    Dimensions:  lat
    Attributes:
     _FillValue           = NaN
     long_name            = Latitude

  lon   (480)
    Datatype:    Union{Missing, Float32} (Float32)
    Dimensions:  lon
    Attributes:
     _FillValue           = NaN
     long_name            = Longitude

  quantiles   (19)
    Datatype:    Union{Missing, Float64} (Float64)
    Dimensions:  quantiles
    Attributes:
     _FillValue           = NaN
     long_name            = Quantiles

  vals   (19 × 480 × 258 × 4 × 1)
    Datatype:    Union{Missing, Float32} (Float32)
    Dimensions:  quantiles × lon × lat × lead × forecast_date
    Attributes:
     _FillValue           = NaN
     long_name            = Average Temperature
     units                = degC

  anom   (19 × 480 × 258 × 4 × 1)
    Datatype:    Union{Missing, Float32} (Float32)
    Dimensions:  quantiles × lon × lat × lead × forecast_date
    Attributes:
     _FillValue           = NaN
     long_name            = Average Temperature Anomaly
     units                = degC

  forecast_period   (2 × 4 × 1)
    Datatype:    DateTime (Int64)
    Dimensions:  nbnds × lead × forecast_date
    Attributes:
     long_name            = Forecast Validity Period
     units                = days since 2024-04-01 00:00:00
     calendar             = proleptic_gregorian

  lead_bnds   (2 × 4)
    Datatype:    Int64 (Int64)
    Dimensions:  nbnds × lead
    Attributes:
     long_name            = Lead Bounds

Global attributes
  long_name            = Average Temperature
  units                = degC
  clim_period          = ["1990-01-01", "2019-12-31"]
  coordinates          = forecast_period
```

**Comment by lazarusA** [comment]
closed by #480
